Dockerfile

```diff
@@ -0,0 +1,213 @@
+# This dockerfile can be built in a few different ways.
```

Comment:
**BUG RISK**
Currently, the project is installed in `/`. This is fine, but it clutters the root directory of the docker image and makes it so that the user cannot simply volume mount their current directory to test its changes via:

```
docker run -ti --rm -v `pwd`:/paramak ukaea/paramak bash
```
If the user tries to volume mount inside of `/`, then they'll replace the entire filesystem.
<p align="right">
    <a href="https://www.pullrequest.com/">
        <img width="15%" src="https://static.pullrequest.com/images/dashboard/pullrequest-logo.svg">
    </a>
</p>